packages/data-store-common/src/__tests__/entities.test.ts

```diff
@@ -0,0 +1,145 @@
+import { Connection, createConnection } from 'typeorm'
+import { ConnectionIdentifierEnum, ConnectionTypeEnum, DataStoreConnectionEntities, DataStoreMigrations } from '../index'
+import { PartyEntity } from '../entities/connection/PartyEntity'
+import { OpenIdConfigEntity } from '../entities/connection/OpenIdConfigEntity'
+import { DidAuthConfigEntity } from '../entities/connection/DidAuthConfigEntity'
+import { ConnectionEntity, connectionEntityFrom } from '../entities/connection/ConnectionEntity'
+
+describe('Database entities test', () => {
+  let dbConnection: Connection
+  const connection_relations = ['config', 'metadata', 'identifier']
+
+  beforeEach(async () => {
+    dbConnection = await createConnection({
+      type: 'sqlite',
+      database: ':memory:',
+      // logging: 'all',
+      migrationsRun: false,
+      migrations: DataStoreMigrations,
+      synchronize: false,
+      entities: DataStoreConnectionEntities,
+    })
+    // await dbConnection.dropDatabase()
```

Comment:
this can be removed?

Reply:
Yeah will remove the dropDatabase. If you don't mind I will leave the logging comment. This allows someone to enable it quickly in case of problems with the test